app/presenters/tree_builder_genealogy.rb

```diff
@@ -38,9 +38,9 @@ def vm_icon_image(vm)
     if vm.template?
       {:icon => "product product-template"}
     elsif vm.retired
-      {:image => '100/retired.png'}
+      {:image => 'svg/currentstate-retired.svg'}
     else
-      {:image => "100/#{vm.current_state.downcase}.png"}
+      {:image => "svg/#{vm.current_state.downcase}.svg"}
```

Comment:
looks like thise should be `svg/currentstate-#{vm.current_state.downcase}.svg`, right?

At least, I don't see `svg/off.svg` but do see `svg/currentstate-off.svg`